components/calendar/src/cal/chinese.rs

```diff
@@ -66,17 +66,18 @@ mod simple;
 /// Júyuè (`M09`), Lùyuè (`M10`), Dōngyuè (`M11`), Bīngyuè (`M12`).
 ///
 /// As a lunar calendar, the lengths of the months depend on the lunar cycle (a month starts on the day of
-/// local new moon), and will be either 29 or 30 days. As the lunar and siderial month do not line up, a leap
+/// local new moon), and will be either 29 or 30 days. As the lunar and solar month do not line up, a leap
```

Comment:
lunar and solar year

Reply:
eh, there's no lunar year and there's no solar month, one has to give. or this needs to be more verbose and say 12 lunar cycles vs one solar cycle/year.

Reply:
"As the calendar year and solar year do not line up", maybe.